2024-10-25 19:29:00,546 ERROR: An error occurred in index route: Could not build url for endpoint 'checkers_search'. Did you mean 'checkers.search' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:108]
2024-10-25 19:29:20,672 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:29:20,673 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:29:21,332 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:29:22,158 ERROR: An error occurred in index route: Could not build url for endpoint 'checkers_search'. Did you mean 'checkers.search' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:108]
2024-10-25 19:29:24,584 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:29:24,584 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:29:25,246 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:29:25,671 ERROR: An error occurred in index route: Could not build url for endpoint 'checkers_search'. Did you mean 'checkers.search' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:108]
2024-10-25 19:29:57,021 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:29:57,022 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:29:57,694 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:29:58,120 ERROR: An error occurred in index route: Could not build url for endpoint 'checkers_search'. Did you mean 'checkers.search' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:108]
2024-10-25 19:30:15,706 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:30:15,707 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:30:16,274 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:30:17,076 ERROR: An error occurred in index route: Could not build url for endpoint 'checkers.checkers_fetch_details'. Did you mean 'checkers.fetch_details' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:108]
2024-10-25 19:30:20,312 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:30:20,313 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:30:20,944 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:30:21,602 ERROR: An error occurred in index route: Could not build url for endpoint 'checkers.checkers_fetch_details'. Did you mean 'checkers.fetch_details' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:108]
2024-10-25 19:32:14,988 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:32:14,989 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:32:15,703 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:32:16,561 ERROR: An error occurred in index route: Could not build url for endpoint 'checkers.checkers_fetch_details'. Did you mean 'checkers.fetch_details' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:108]
2024-10-25 19:32:30,869 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:32:30,870 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:32:31,481 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:32:32,018 ERROR: An error occurred in index route: Could not build url for endpoint 'checkers_fetch_details'. Did you mean 'checkers.fetch_details' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:108]
2024-10-25 19:33:00,289 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:33:00,290 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:33:00,907 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:33:08,370 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:33:08,370 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:33:08,373 INFO: Reset processing flags for SKU 'DJ-0066' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 19:33:09,010 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:33:09,011 INFO: Stale processing flags have been reset. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:123]
2024-10-25 19:33:09,631 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:35:35,771 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:35:35,772 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:35:35,774 INFO: Reset processing flags for SKU 'DC-0002' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 19:35:36,394 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:35:36,395 INFO: Stale processing flags have been reset. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:123]
2024-10-25 19:35:37,063 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:35:42,811 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:35:42,811 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:35:42,814 INFO: Reset processing flags for SKU 'DI-0002' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 19:35:43,509 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:35:43,510 INFO: Stale processing flags have been reset. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:123]
2024-10-25 19:35:42,379 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:35:44,078 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:35:44,078 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:35:44,746 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:36:29,709 INFO: Inventory Validation Startup [in /home/creep/projects/validate-barcode/app/__init__.py:29]
2024-10-25 19:36:41,638 INFO: Inventory Validation Startup [in /home/creep/projects/validate-barcode/app/__init__.py:29]
2024-10-25 19:36:43,191 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:36:43,194 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:36:43,197 INFO: Reset processing flags for SKU 'DA-0262' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 19:36:43,198 INFO: Reset processing flags for SKU 'DB-0005' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 19:36:43,843 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:36:43,845 INFO: Stale processing flags have been reset. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:123]
2024-10-25 19:36:44,503 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 19:38:30,105 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 19:38:30,115 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 19:38:30,118 INFO: Reset processing flags for SKU '13837' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
